paddle/fluid/operators/distributed/grpc_variable_response.cc

```diff
@@ -22,6 +22,9 @@
 #include "paddle/fluid/operators/distributed/grpc_variable_response.h"
 #include "paddle/fluid/platform/profiler.h"
 
+DEFINE_string(rpc_server_profile_path, "/tmp/profile_ps",
```

Comment:
Can't define here since it's useful for both brpc and grpc.

Reply:
done